az-digital/az_quickstart
# Closes #2380 Adding permissions for using shortcuts.

## Description
Gave content admin the following permissions
- 'administer shortcuts'
- 'customize shortcut links'
Gave content editor the following permissions.
- 'switch shortcut sets'

## Related issues
Closes #2380

## How to test
Log in as a user with content administrator role. `az_content_admin`
See if you can edit shortcuts and shortcut sets

Log in as a user with the content editor role. `az_content_editor`
See if you can switch between shortcut sets.

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
